docs/api.md

```diff
@@ -2952,7 +2952,386 @@ HTTP/1.1 200 Success
 
 ## EventCatalogue
 
-> TODO
+`EventCatalogue` maps an emergency response activity to the responsible [party](#core-resources-party), [role](#core-resources-partyrole) or [group](#core-resources-partygroup)
```

Comment:
Party, Role, Group should be in caps. eg PARTY, ROLE, GROUP

Reply:
Done